packages/utilities/src/lib/objectValues.ts

```diff
@@ -0,0 +1,5 @@
+import type { NonNullObject } from './utilityTypes';
+
+export function objectValues<T extends NonNullObject>(obj: T): T[keyof T][] {
+	return Object.values(obj) as T[keyof T][];
```

Comment:
The typing isn't correct for array
![image](https://user-images.githubusercontent.com/74945038/193395583-f5b45bfa-ba5b-4bd2-8cb2-5c037c91cd32.png)